src/main/java/com/nerodesk/om/mock/MkAttributes.java

```diff
@@ -89,11 +91,37 @@ public Date created() throws IOException {
 
     @Override
     public boolean visible() throws IOException {
-        return this.shown;
+        final boolean shown;
+        if (this.file.exists()) {
```

Comment:
@krzyk This is just for `MkAttributes`. I actually did it that way at first but a lot of our mock classes will suddenly become broken. That's why we had the `exists()` check.